tests/unit/r2i/tensorflow/engine.cc

```diff
@@ -62,15 +62,25 @@ ImageFormat Frame::GetFormat () { return ImageFormat::Id::UNKNOWN_FORMAT; }
 std::shared_ptr<TF_Tensor> Frame::GetTensor (std::shared_ptr<TF_Graph> graph,
     TF_Operation *operation, RuntimeError &error) { return nullptr; }
 
-Prediction::Prediction () {}
-double Prediction::At (unsigned int index,  r2i::RuntimeError &error) { return 0.0; }
-void *Prediction::GetResultData () { return nullptr; }
-unsigned int Prediction::GetResultSize () { return 0; }
-RuntimeError Prediction::SetTensor (std::shared_ptr<TF_Graph> graph,
-                                    TF_Operation *operation, std::shared_ptr<TF_Tensor> tensor) { return RuntimeError(); }
+float *Engine::GetTensorData(TF_Operation *operation,
+                             std::shared_ptr<TF_Tensor> tensor, RuntimeError &error) { return nullptr; }
+int64_t Engine::GetRequiredBufferSize (std::shared_ptr<TF_Graph> pgraph,
+                                       TF_Operation *operation, int index, RuntimeError &error) { return 0; }
 }
 }
 
+namespace r2i {
+
+double Prediction::At (unsigned int output_index, unsigned int index,
+                       r2i::RuntimeError &error) { return 0.0; }
+void *Prediction::GetResultData (unsigned int output_index,
+                                 RuntimeError &error) { return nullptr; }
+unsigned int Prediction::GetResultSize (unsigned int output_index,
+                                        RuntimeError &error) { return 0; }
+RuntimeError Prediction::AddResult (float *data, unsigned int size) { return RuntimeError(); }
+
+}
+
```

Comment:
Does the new code in Engine require new tests? Seems like they are missing.

Reply:
Added